### PR TITLE
Version bump to 1.4.0

### PR DESCRIPTION
**Pull request by lukecarbis**
This release applies some finishing touches to the repeater field. It also introduces a new "Field Width" feature, which lets you choose the width of the fields as seen in the Editor.

* New: Function to reset repeater rows: reset_block_rows()
* New: Add a "Field Width" control to blocks
* Fix: Empty repeater rows now save and can be moved properly
* Fix: An issue which occasionally prevented repeater rows from being deleted
* Fix: Prevent repeated requests to validate a Pro license
* Tweaks: Add a different admin notice for when a license validation request fails
* Tweaks: Many new and shiny unit and integration tests, making Block Lab more solid than ever

### CODE REVIEW

**Comment by kienstra** [comment]
@lukecarbis,
This looks good, having tested all fields, and all fields in a repeater. 

It looks like the `Textarea` control has the same issue was saw in #409 with the repeater. If it's alright, I'm going to work on a PR for that.

**Comment by kienstra** [comment]
Also, what do you think about [creating a GitHub release](https://github.com/getblocklab/block-lab/releases/new) for `v1.4.0` , once this PR is merged? Your [comment above](https://github.com/getblocklab/block-lab/pull/413#issue-3138207650) would be perfect for the 'Describe this release section,' and it could just target the `develop` branch.

**Comment by lukecarbis** [comment]
Sounds good! I'll hold off merging this until we get the PR merged for the `Textarea` bug you found.

**Comment by kienstra** [comment]
Sure, #414 addresses the issue with the Textarea.

**Comment by lukecarbis** [comment]
Okay, that fix is now included in this version bump PR.

**Review by kienstra — APPROVED** [review]

**Comment by kienstra** [comment]
This looks good!

**Pull request merged**
